io.edgehog.devicemanager.apps.ApplicationCommand.json

```
@@ -0,0 +1,18 @@
+{
+  "interface_name": "io.edgehog.devicemanager.apps.ApplicationCommand",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "server",
+  "aggregation": "individual",
```

It would be nice to have a small description of each interface:
```suggestion
  "aggregation": "individual",
  "description": "Possible commands sent to a device to start an operation on an application",
```